src/environments/array.js

```diff
@@ -48,11 +52,12 @@ function getHLines(parser: Parser): boolean[] {
  */
 function parseArray(
     parser: Parser,
-    {hskipBeforeAndAfter, addJot, cols, arraystretch}: {|
+    {hskipBeforeAndAfter, addJot, cols, arraystretch, colSeparationType}: {|
         hskipBeforeAndAfter?: boolean,
         addJot?: boolean,
         cols?: AlignSpec[],
         arraystretch?: number,
+        colSeparationType?: number,
```

Comment:
```suggestion
        colSeparationType?: ColSeparationType,
```

Reply:
Is this needed? It seems there is no `parseArray` calls with `colSeparationType`.